app/models/audit/event/api_key.rb

```diff
@@ -0,0 +1,91 @@
+module Audit
+  module Event
+    class ApiKey
+
+      def initialize(auth_role:, subject_role:, success:, error_message: nil)
+        @auth_role = auth_role
+        @subject_role = subject_role
```

Comment:
This should take the `role_id` instead of the role, since we only ever use the `role_id`.  As a side benefit, we'll make reek happy.
<img width="332" alt="happyreek" src="https://user-images.githubusercontent.com/699684/84935046-2d57a600-b0a6-11ea-9a22-0cce94d1afa7.png">